torch/_torch_docs.py

```diff
@@ -7859,7 +7859,7 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.sort,
            r"""
-sort(input, dim=-1, descending=False, *, out=None) -> (Tensor, LongTensor)
+sort(input, dim=-1, descending=False, stable=False, *, out=None) -> (Tensor, LongTensor)
 
```

Comment:
Probably it makes sense to move ` *` before `stable=False`.

Reply:
That sounds reasonable, too.